oneoffs/cleanup_audits.py

```diff
@@ -1,19 +1,21 @@
 """
 There was an issue that allowed duplicate audits to be created
-This will cleanup all the ones due on the specific date:
+This will cleanup all the ones due on the specific date
 Run with: grouper-ctl -v oneoff run AuditCleanup --no-dry_run
 """
 import logging
 from datetime import datetime
 
 from grouper.models.audit import Audit
+from grouper.models.audit_member import AuditMember
 from grouper.oneoff import BaseOneOff
 
 
 class AuditCleanup(BaseOneOff):
     def run(self, session, dry_run=True):
         # Pull the audits that are expiring on 2020-03-13 00:00:00. None of these have been
-        # completed. About half of the ones for the 31st have already been completed
+        # completed as of my checks. About half of the ones for the 31st have already
+        # been completed
```

Comment:
Nit: Preference to just say that the hardcoded 2020-03-13 is the date we had to delete when this happened, rather than baking in additional Dropbox-specific and right-now-specific info about completion progress.